components/brave_rewards/core/legacy/media/youtube.cc

```diff
@@ -25,6 +25,28 @@ using std::placeholders::_3;
 
 namespace brave_rewards::internal {
 
+namespace {
+
+bool getJSONValue(const std::string& field_name,
```

Comment:
Moved from `bat_helper.cc` so that that file could be removed without affecting this one.